core/aws_ddk_core/cicd/pipeline.py

```diff
@@ -320,6 +324,47 @@ def add_test_stage(
         )
         return self
 
+    def add_notifications(
+        self,
+        notification_rule: Optional[NotificationRule] = None,
+    ) -> "CICDPipelineStack":
+        """
+        Add pipeline notifications. Create notification rule that sends events to the specified SNS topic.
+
+        Parameters
+        ----------
+        notification_rule: Optional[NotificationRule]
+            Override notification rule
+
+        Returns
+        -------
+        pipeline : CICDPipeline
+            CICD pipeline
+        """
+
+        self._notification_rule = notification_rule or NotificationRule(
+            self,
+            "notification",
+            detail_type=DetailType.BASIC,
+            events=["codepipeline-pipeline-pipeline-execution-failed"],
+            source=self._pipeline.pipeline,
+            targets=[
+                Topic.from_topic_arn(
+                    self, "topic", topic_arn=self._config.get_env_config("cicd").get("notifications_topic_arn")
+                )
+                if self._config.get_env_config("cicd").get("notifications_topic_arn")
+                else Topic(
+                    self,
+                    f"{self.pipeline_name}-cicd-notifications",
+                    topic_name=f"{self.pipeline_name}-cicd-notifications",
+                    master_key=Key.from_lookup(
+                        self, f"{self.pipeline_name}-cicd-notifications-key", alias_name="alias/aws/sns"
```

Comment:
Nice! Does that mean that there will always be a CMK in an AWS account with the alias `alias/aws/sns`? No need to create it?